FirebaseMessaging.podspec

```diff
@@ -35,15 +35,22 @@ device, and it is completely free.
 
   base_dir = "FirebaseMessaging/"
   s.source_files = [
-    base_dir + 'Sources/**/*.[mh]',
+    base_dir + 'Sources/**/*',
+    base_dir + 'Sources/Protogen/nanopb/*.h',
     'Interop/Analytics/Public/*.h',
     'FirebaseCore/Sources/Private/*.h',
     'FirebaseInstallations/Source/Library/Private/*.h',
+    'GoogleDataTransport/Public/*.h',
```

Comment:
Can this line be removed now since `GoogleDataTransport` has been added as a dependency on line 67?

Reply:
I'm not following, so do analytics and installations which had their headers added here.

Reply:
Interop/Analytics/Public and FirebaseInstallations/Source/Library are part of this GitHub repo. GoogleDataTransport is not.

Reply:
Done.